### PR TITLE
Richard_FindOutlier

### PR DESCRIPTION
**Pull request by RichardSeverich**

### CODE REVIEW

File: src/main/java/org/fundacionjala/coding/richard/FindOutlier.java [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+     */
+    public FindOutlier() {
+    }
+
+    /**
+     * @param outlier is a array of int numbers.
+     * @return outlier number.
+     */
+    int find(int[] outlier) {
+        int pairResult = 0;
```
Comment (carledriss) [inline src/main/java/org/fundacionjala/coding/richard/FindOutlier.java]: pair ?
even means par
odd means impar
:P

File: src/main/java/org/fundacionjala/coding/richard/FindOutlier.java [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+ * Should return: 11
+ * [160, 3, 1719, 19, 11, 13, -21]
+ * Should return: 160
+ */
+public class FindOutlier {
+
+    /**
+     * This method is a private constructor.
+     */
+    public FindOutlier() {
```
Comment (carledriss) [inline src/main/java/org/fundacionjala/coding/richard/FindOutlier.java]: Remove this unnecessary constructor.

Comment (RichardSeverich) [reply src/main/java/org/fundacionjala/coding/richard/FindOutlier.java]: okey  I remove it.

File: src/main/java/org/fundacionjala/coding/richard/FindOutlier.java (hunk 1 above)
Comment (RichardSeverich) [reply src/main/java/org/fundacionjala/coding/richard/FindOutlier.java]: Yes.

Comment (RichardSeverich) [reply src/main/java/org/fundacionjala/coding/richard/FindOutlier.java]: Okay, already I have solved that.

**Comment by carledriss** [comment]
LGTM!

**Pull request merged**